Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift

```diff
@@ -380,6 +380,22 @@ extension VariableDeclSyntax {
   }
 }
 
+// MARK: Initializing collections from protocols
+// These initializers allow the creation of syntax collections that have a base
+// node as their element from the corresponding protocol type.
+
+extension ExprListSyntax {
+  public init(_ elements: [ExprSyntaxProtocol]) {
+    self.init(elements.map { ExprSyntax(fromProtocol: $0) } as [ExprSyntax])
+  }
+}
+
+extension UnexpectedNodesSyntax {
+  public init(_ elements: [SyntaxProtocol]) {
+    self.init(elements.map { Syntax(fromProtocol: $0) } as [Syntax])
+  }
+}
+
```

Comment:
Let’s keep them in alphabetical order. AFAICT `TupleExprElement` is the only extension that’s not currently in alphabetical order (I think because it was renamed from `TupleExprElement`). If you want, you can move that as well to restore alphabetical order.